### PR TITLE
Melvin Cedeno  morse_translator

### PR DESCRIPTION
**Pull request by TheCraftedGem**

### CODE REVIEW

File: lib/morse_translator.rb [hunk 1]
```diff
@@ -1,43 +1,94 @@
...
+        "R" => ".-.",
+        "S" => "...",
+        "T" => "-",
+        "U" => "..-",
+        "V" => "...-",
+        "W" => ".--",
+        "X" => "-..-",
+        "Y" => "-.--",
+        "Z"  => "--..",
+        "a" => ".-",
```
Comment (memcmahon) [inline lib/morse_translator.rb:33]: This is VERY repetitive!  Can you refactor this to work without duplicating the alphabet in the dictionary?

File: lib/morse_translator.rb [hunk 2]
```diff
@@ -1,43 +1,94 @@
...
+        "6" => "-....",
+        "7" => "--...",
+        "8" => "---..",
+        "9" => "----.",
+        "0" => "-----",
+        " " => " "}
+                    
+  end
+
+  def create_upper_case_keys
```
Comment (memcmahon) [inline lib/morse_translator.rb:73]: It doesn't look like you are using this method anywhere? Why is this here?

File: lib/morse_translator.rb [hunk 3]
```diff
@@ -1,43 +1,94 @@
...
+    @dictionary.keys.map do |element|
+      element.upcase
+    end
+  end
+
+  def translate(input)
+    input_array = input.chars
+    input_array.map do |keys|
+      @dictionary.values_at(keys)
```
Comment (memcmahon) [inline lib/morse_translator.rb:82]: This works, but there is an easier way to access a value from a specific key: `@dictionary[key]`

File: test/morse_translator_test.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/morse_translator'
+
+class MorseTranslatorTest < Minitest::Test
```
Comment (memcmahon) [inline test/morse_translator_test.rb:5]: Where is your test for `create_upper_keys`?

**Review by memcmahon — COMMENTED** [review]
It looks like you need to spend some time reviewing the basics of Hash manipulation - I would have liked to see you use the hash[key] notation to access the value you are looking for.

File: lib/morse_translator.rb (hunk 2 above)
Comment (TheCraftedGem) [reply lib/morse_translator.rb:73]: I used it to create the upcase keys then just added em to the dictionary.
